recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
```

Comment:
@ReimarBauer can you try a minimum example that show the issues? I try:

```shell
source activate TEST
conda create netcdf4 -n TEST -c conda-forge --yes
python -c "import netCDF4; print(netCDF4.__version__)"
```

and everything works. If you are able to reproduce the error send me the results for:

```shell
conda info
conda list
```